hackathon_site/dashboard/frontend/yarn.lock

```diff
@@ -9028,9 +9028,9 @@ path-key@^3.0.0, path-key@^3.1.0:
   integrity sha512-ojmeN0qd+y0jszEtoY48r0Peq5dwMEkIlCOu6Q5f41lfkswXuKtYrhgoTpLnyIcHm24Uhqx+5Tqm2InSwLhE6Q==
 
 path-parse@^1.0.6:
```

Comment:
I think when you were resolving merge conflicts, you kept your version of this file instead of the one from develop. Can you restore this file back to what's on develop?

```
$ git checkout develop -- hackathon_site/dashboard/frontent/yarn.lock
```